### ISSUE TITLE
Publisher City required for legal papers

### ISSUE THREAD

**Issue by colomet** [pressbooks/ideas#131]
### Feature Description

To show Publisher city in the cover page

### Feature Use Case

I´m asking the ISSN number in the spanish books office. And they require to show the publisher city in the cover page ir order to give the ISSN number.

(ISSN is the number I need as my books keeps updating often)

**Comment by colomet** [comment]
Maybe inside of the license information (at Book Information in the cover section)

**Comment by SteelWagstaff** [comment]
@colomet the publisher city is currently included on the 'title page' for book exports (this is true for both PDFs and EPUBs for instance). Are you asking to have it included with the book metadata on the webbook's homepage as well?

**Comment by colomet** [comment]
Yes, it must be in the web site or i can not have the issn number for the site.

thanks.
